include/rgl/api/core.h

```diff
@@ -275,7 +275,7 @@ rgl_cleanup(void);
 /******************************** MESH ********************************/
 
 /**
- * Creates mesh from vertex and index arrays. CW/CCW order does not matter.
+ * Creates mesh from a vertex and index arrays. CW/CCW order does not matter.
```

Comment:
To avoid the confusion of plurals:
"Creates mesh from the arrays of vertices and indices."